src/test/regress/expected/pg12.out

```diff
@@ -0,0 +1,90 @@
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+')::int > 11 AS server_version_above_eleven;
+ server_version_above_eleven
+-----------------------------
+ t
+(1 row)
+
+create schema test_pg12;
+set search_path to test_pg12;
+CREATE FUNCTION blackhole_am_handler(internal)
+RETURNS table_am_handler
+AS 'citus'
+LANGUAGE C;
+CREATE ACCESS METHOD blackhole_am TYPE TABLE HANDLER blackhole_am_handler;
+create table test_am(id int, val int) using blackhole_am;
+insert into test_am values (1, 1);
+-- Custom table access methods should be rejected
+select create_distributed_table('test_am','id');
+ERROR:  only heap AM is supported
+-- Test generated columns
+create table gen1 (
+	id int,
+	val1 int,
+	val2 int GENERATED ALWAYS AS (val1 + 2) STORED
+);
+create table gen2 (
+	id int,
+	val1 int,
+	val2 int GENERATED ALWAYS AS (val1 + 2) STORED
+);
+insert into gen1 (id, val1) values (1,4),(3,6),(5,2),(7,2);
+insert into gen2 (id, val1) values (1,4),(3,6),(5,2),(7,2);
+select * from create_distributed_table('gen1', 'id');
```

Comment:
I think if `create_distributed_table ` errors out, it'd be a nicer user experience. As we do `ERROR:  only heap AM is supported`